ComprefaceTestApp/DTOs/ExampleSubject/DeleteImageById/DeleteImageByIdRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace ComprefaceTestApp.DTOs.ExampleSubject.DeleteImageById
+{
+    public class DeleteImageByIdRequest
+    {
+        [JsonPropertyName("image_id")]
```

Comment:
without specifying the `JsonPropertyName` Json deserialization should also work so just remove this attribute, there already setup for deserialization from snake case to camel case on a global settings in `Program.cs Line 45`